src/components/common/WalletConnectModal.svelte

```diff
@@ -26,6 +26,7 @@
             );
           }}
           disabled={loading}
+          class="plausible-event-name=Button+Close+Button"
```

Comment:
let's make this more specific, ex: `Button+Close+WalletConnect+Modal`

Reply:
done